### ISSUE TITLE
Installation error on Windows/WSL

### ISSUE THREAD

**Issue by dennisschott** [eliassjogreen/denon#42]
Hey guys, i have an issue while installing denon. I installed deno (v1.0.1) some minutes ago and now i've tried to install denon. I get this error on Windows and Windows WSL Ubuntu:

....
Compile https://deno.land/x/denon/denon.ts
error: TS2345 [ERROR]: Argument of type '"file" | "dir" | "symlink" | undefined' is not assignable to parameter of type 'SymlinkOptions | undefined'.
  Type '"file"' is not assignable to type 'SymlinkOptions | undefined'.
  await Deno.symlink(originSrcFilePath, dest, type);
                                              ~~~~
    at https://deno.land/std@v0.51.0/fs/copy.ts:114:47

TS2345 [ERROR]: Argument of type '"file" | "dir" | "symlink" | undefined' is not assignable to parameter of type 'SymlinkOptions | undefined'.
  Type '"file"' is not assignable to type 'SymlinkOptions | undefined'.
  Deno.symlinkSync(originSrcFilePath, dest, type);
                                            ~~~~
    at https://deno.land/std@v0.51.0/fs/copy.ts:132:45

TS2345 [ERROR]: Argument of type '"file" | "dir" | "symlink" | undefined' is not assignable to parameter of type 'SymlinkOptions | undefined'.
  Type '"file"' is not assignable to type 'SymlinkOptions | undefined'.
  await Deno.symlink(src, dest, srcFilePathType);
                                ~~~~~~~~~~~~~~~
    at https://deno.land/std@v0.51.0/fs/ensure_symlink.ts:31:33

TS2345 [ERROR]: Argument of type '"file" | "dir" | "symlink" | undefined' is not assignable to parameter of type 'SymlinkOptions | undefined'.
  Type '"file"' is not assignable to type 'SymlinkOptions | undefined'.
  Deno.symlinkSync(src, dest, srcFilePathType);
                              ~~~~~~~~~~~~~~~
    at https://deno.land/std@v0.51.0/fs/ensure_symlink.ts:58:31

Found 4 errors.

**Comment by dennisschott** [comment]
It seems that it's an issue with version 1.0.1. On version 1.0.0 it works fine.

**Comment by ansonfoong** [comment]
I am also experiencing this issue, was using rca version and upgraded to 1.0.1 and the error came up when I tried running the denon command as well as installing.

**Comment by Qu4k** [comment]
This has been fixed in 1a64e85269e9ad8fa79e026bdd6981ea330b5c4d on the `master` branch and d3e964d4f8aaf8fc666260ba5a856972e8a5d399 on the `rework` branch. Deno 1.0.1 introduced breaking changes to the filesystem standard library.
